internal/cloud/realm/realm.go

```diff
@@ -79,6 +79,45 @@ var (
 		LocationMumbai.String(),
 		LocationSingapore.String(),
 	}
+	ProviderRegionValues = []string{
+		AWSProviderRegionUSEast1.String(),
+		AWSProviderRegionUSWest2.String(),
+		AWSProviderRegionEUCentral1.String(),
+		AWSProviderRegionEUWest1.String(),
+		AWSProviderRegionAPSoutheast1.String(),
+		AWSProviderRegionAPSoutheast2.String(),
+		AWSProviderRegionAPSouth1.String(),
+		AWSProviderRegionUSEast2.String(),
+		AWSProviderRegionEUWest2.String(),
+		AWSProviderRegionSAEast1.String(),
+		AzureProviderRegionEastUS2.String(),
+		AzureProviderRegionWestUS.String(),
+		AzureProviderRegionWestEurope.String(),
+		AzureProviderRegionEastAsia.String(),
+		AzureProviderRegionSouthEastAsia.String(),
+	}
+	ProviderRegionValuesByCloudProvider = map[string][]string{
+		CloudProviderAWS: {
+			AWSProviderRegionUSEast1.Label(),
+			AWSProviderRegionUSWest2.Label(),
+			AWSProviderRegionEUCentral1.Label(),
+			AWSProviderRegionEUWest1.Label(),
+			AWSProviderRegionAPSoutheast1.Label(),
+			AWSProviderRegionAPSoutheast2.Label(),
+			AWSProviderRegionAPSouth1.Label(),
+			AWSProviderRegionUSEast2.Label(),
```

Comment:
These are new and currently unsupported. However given the timelines, I think we should just add GCP in this mapping as well. I think by the time we release this, we should be ready in the newest regions.

Reply:
I agree, but for now I left a TODO I can fill out/remove once we sync up on when we want the cli released